awx/main/access.py

```diff
@@ -1339,7 +1339,7 @@ def can_add(self, data):
     def can_change(self, obj, data):
         if obj and obj.organization_id is None:
             raise PermissionDenied
-        if self.user not in obj.organization.execution_environment_admin_role:
+        if self.user not in obj.organization.execution_environment_admin_role and self.user not in obj.organization.admin_role:
```

Comment:
It turns out that this change isn't necessary:

    In [15]: pprint([(u, u in o.execution_environment_admin_role) for u in User.obje
        ...: cts.all()])
    [(<User: admin>, True),
     (<User: auditor>, False),
     (<User: org-admin>, True),
     (<User: org-ee-admin>, True),
     (<User: org-member>, False),
     (<User: ordinary>, False)]

Reply:
alright, removed